content/en/tracing/trace_collection/proxy_setup/apigateway.md

```diff
@@ -37,66 +57,109 @@ Datadog APM supports tracing for API Gateway when API Gateway proxies requests t
-          "integration.request.header.x-dd-proxy-request-time-ms": "context.requestTimeEpoch",
-          "integration.request.header.x-dd-proxy-domain-name": "context.domainName",
-          "integration.request.header.x-dd-proxy-httpmethod": "context.httpMethod",
-          "integration.request.header.x-dd-proxy-path": "context.path",
-          "integration.request.header.x-dd-proxy-stage": "context.stage",
-        }
-      }
-    })
-    # other settings here
-    });
-   ```
-   {{% /tab %}}
-   
-   {{% tab "AWS Console" %}}
-   
-   1. In the AWS Management Console, navigate to API Gateway and go to your API's **Resources** page. 
-
-   2. Go to **Integration request** and click **Edit**.
-
-   3. Under **Edit integration request**, go to **URL request headers parameters**. Click **Add request header parameter**.
+API Gateway must pass the following headers in the request for the tracer to create the inferred span:
+| header | value |
+| ------ | ----- |
+| `x-dd-proxy` | `'aws-apigateway'` |
+| `x-dd-proxy-request-time-ms` | `context.requestTimeEpoch` |
+| `x-dd-proxy-domain-name` | `context.domainName` |
+| `x-dd-proxy-httpmethod` | `context.httpMethod` |
+| `x-dd-proxy-path` | `context.path` |
+| `x-dd-proxy-stage` | `context.stage` |
+
+To pass the headers, complete the following steps for your API type:
+
+### REST API (V1)
+
+To pass in the required headers, you can use the AWS CDK or AWS Console.
+
+{{< tabs >}}
+{{% tab "AWS CDK" %}}
+
+Add the headers under `requestParameters` and use `$context` variables:
+
+```
+const resource = api.root.addResource('myresource');
+ resource.addMethod('ANY', new apigateway.Integration({
+   # other configurations
+   options: {
+     # other options
+     requestParameters: {
+       "integration.request.header.x-dd-proxy": "'aws-apigateway'",
+       "integration.request.header.x-dd-proxy-request-time-ms": "context.requestTimeEpoch",
+       "integration.request.header.x-dd-proxy-domain-name": "context.domainName",
+       "integration.request.header.x-dd-proxy-httpmethod": "context.httpMethod",
+       "integration.request.header.x-dd-proxy-path": "context.path",
+       "integration.request.header.x-dd-proxy-stage": "context.stage",
+     }
+   }
+ })
+ # other settings here
+ });
+```
+{{% /tab %}}
+
+{{% tab "AWS Console" %}}
+
+1. In the AWS Management Console, navigate to API Gateway and go to your API's **Resources** page. 
+
+2. Go to **Integration request** and click **Edit**.
+
+3. Under **Edit integration request**, go to **URL request headers parameters**. Click **Add request header parameter**.
 
 {{< img src="tracing/trace_collection/apigateway/console_headers.png" alt="Your HTTP headers for your API in API Gateway, after you have added all six header parameters." style="width:100%;" >}}
 
-   {{% /tab %}}
-   {{< /tabs >}}
+{{% /tab %}}
+{{< /tabs >}}
+
+### HTTP API (V2)
+
+Attach the parameter mapping that injects the headers.
```

Comment:
Can we also include here info on what customers will need to do this manually, should they not deploy using the cdk? I'm thinking something along the lines of the "header|value" table that's included in V1. You can see what exactly those are [here](https://github.com/DataDog/datadog-cdk-constructs/blob/b7a9aa75a8d40a89718481fecf8f41e175f5704f/src/apigateway/parameters.ts#L16-L23).

It's vitally important that the `x-dd-proxy-request-time-ms` gets three zeros appended, as done with `"${context.requestTimeEpoch}000"`, which turns the request time from seconds to milliseconds. Without this, the start time of the synthetic span will be off my a factor of 1000.